pacu/modules/iam__privesc_scan/main.py

```diff
@@ -450,21 +453,25 @@ def main(args, pacu_main: 'Main'):
                             if role_escalation_methods[method][permission] is True:  # If the permission is required for the method
                                 permission = permission.lower()
                                 if permission in role['Permissions']['Deny']:
-                                    is_possible = False
-                                    break
+                                    if 'IfResourcesNotIn' in str(role['Permissions']['Deny'][permission]["Conditions"]):
```

Comment:
This is the same logic as [line 421](https://github.com/RhinoSecurityLabs/pacu/pull/344/commits/9e2c6dc1695ed77719cb82a527574523a8c2bf96#diff-9a7bf423906850ad049f2a6f5a63f92ffce10ec7bd788825168ac6d8deb28f00R421). I'd suggest making this a function and calling it in both locations.